v2/blockstore/readonly.go

```diff
@@ -320,6 +348,16 @@ func (b *ReadOnly) GetSize(key cid.Cid) (int, error) {
 	return fnSize, nil
 }
 
+func isIdentity(key cid.Cid) (ok bool, digest []byte, err error) {
```

Comment:
nit: usually the value would go before `ok bool`, and an error always goes last, so: `digest []byte, ok bool, err error`